### ISSUE TITLE
SKR 1.4 Turbo blown component

### ISSUE THREAD

**Issue by exter1312** [bigtreetech/BIGTREETECH-SKR-V1.3#556]
Hey, i changed the Creality Board on my Ender 5 Plus to a SKR 1.4 Turbo and was using it with Klipper.
It worked flawlessly until a couple of minutes ago when i saw a bright yellow flash inside the housing.

![skr1 4](https://user-images.githubusercontent.com/95894960/145507667-010b31db-2074-4fd5-b73a-70c968f29bf8.jpg)

![1 4](https://user-images.githubusercontent.com/95894960/145507682-4ad528e8-2701-419c-bd3f-523150d060b0.jpg)

Upon further inspection i found where the flash might have come from.
I dont know what happend, since it blew mid-print.
I hope somebody can tell me what happend.

The board seems to be working when powered by USB only, but as soon as i switch the jumper to 5v VDD only the fans start working.

**Comment by exter1312** [comment]
Ok i looked at the schematics and it seems to be C26 used for 5v DCDC circuit. I am still unsure about the reason why it happened.
![circ](https://user-images.githubusercontent.com/95894960/145559619-36f140f8-953d-448e-bfdb-098ad9efc049.PNG)

**Comment by radek8** [comment]
It's pretty weird.
The SKR1.4 board can supply 12-24V.
If I power it 24V then DCIN will be 24V. after voltage drop on diode V1 the voltage will be VIN = 23.5V
The VIN voltage supplies this capacitor, but according to the documentation it is only dimensioned for 10V (22UF / 10V).
It looks like a schematic error or an undersized capacitor that could explode for this reason.
I would try to desolder it, clean the surroundings, repair the connections and replace it with a new one, dimensioned at> 25V.
I have the same board and the same 24V power supply ... I think a nasty surprise awaits me :-(

![image](https://user-images.githubusercontent.com/46979052/145624014-2b142703-704b-4626-a6c6-3316ec5ad88a.png)

![image](https://user-images.githubusercontent.com/46979052/145623892-6ebc80ec-57e3-42b3-a348-4cd8795e2714.png)

**Comment by radek8** [comment]
I would solve the problem with an external 5V power supply module which is supplied with this board as an optional accessory.
Remove jumpr DC SEL !!! And connected all 3 pins on connector J9
It will work and without the need for soldering.

![image](https://user-images.githubusercontent.com/46979052/145626158-d490680c-00c3-433c-a2ec-30721a5b65d5.png)

**Comment by exter1312** [comment]
Thank you for your reply.
Yes i thought about the DCDC extensionboard. Will it just replace the onboard circuit? Because replacing the capacitor will not be possible since it took out the solderpads and the underlaying copper. I will give it a shot. 
I also contacted BTT about this issue. 
OK  you are talking about the NPWR jumpers? Do i need to bridge all three of them?

**Comment by radek8** [comment]
Yes, the external module replaces the on-board power supply.
However, you must completely remove the DC SEL connection in case the on-board power supply is damaged.
And connect all 3 pins together on connector J9 - NPWR. This will send 5V from the External Source to the branch of the original 5V branch.

**Comment by radek8** [comment]
I would not rely too much on contacting BTT. According to user reactions, they do not communicate much. Complaints do not solve. If they answer, they will send you to the merchant from whom you bought the board .... 
:(

**Comment by exter1312** [comment]
Yeah sadly i know. I bought it directly from them through aliexpress. Maybe i can get the money back though.

**Comment by radek8** [comment]
Just look at the problems with the SKR2 board.
They didn't worry much about it.

**Comment by exter1312** [comment]
Ok if i understood you correctly i have to bridge these three pins.
Thank you very much for your help. 


![192932](https://user-images.githubusercontent.com/95894960/145632694-04d58815-b7c2-425a-8eb2-2c806665b856.PNG)

**Comment by radek8** [comment]
> Yeah sadly i know. I bought it directly from them through aliexpress. Maybe i can get the money back though.

The merchant will not return the money to you alone, only Aliexperss would intervene.
And if it's been over a month, Aliexperss won't deal with it

**Comment by radek8** [comment]
Yes

![image](https://user-images.githubusercontent.com/46979052/145633312-bd8f79c5-2046-434e-8cca-8ba5a82c5719.png)

**Comment by exter1312** [comment]
If everything else fails i will try to also get paypal involved.

Thanks again. I will order the DCDC extension and keep this thread updated.

**Comment by radek8** [comment]
Unplug this one completely


![image](https://user-images.githubusercontent.com/46979052/145633748-a21339a4-a4fe-49fb-a27d-13328ffbf554.png)

**Comment by exter1312** [comment]
Alright. Everything should be here by Tuesday/Wednesday.

**Comment by exter1312** [comment]
Update: It seems to work sofar.

The printer is up and running. Nothing exploded until yet. 
BTT wrote me through aliexpress and tried to give me a little support. They told me to check my wiring. Haha.

What ever, maybe they'll give me some kind of store discount.

Thanks again so much.

**Comment by radek8** [comment]
With that 5V power module?

**Comment by exter1312** [comment]
Yes. I soldered the NPWR pins on J9 together and removed the DC Selector jumper.

Finally i can use stealthchop on the TMC2209 and enjoy the silence.

**Comment by radek8** [comment]
TMC2209 I  have too, they are great.

**Comment by radek8** [comment]
Be careful to have the TMC 2209 switched to UART mode. Often they sell them without a jumper

**Comment by exter1312** [comment]
Thanks.
I have the BTT TMC 2209 v1.2. I believe they are already soldered for UART. I just removed the pin for sensorless homing.
Absolutely quiet.
Only the PSU fan is driving me crazy. Maybe i can swap it, or throttle it with resistors.

**Comment by radek8** [comment]
Probably the same problem.
Maybe the wrong batch
https://github.com/bigtreetech/BIGTREETECH-SKR-V1.3/issues/552

**Comment by radek8** [comment]
Please close the ticket
